caffe2/python/pybind_state.cc

```diff
@@ -1186,11 +1187,17 @@ void addGlobalMethods(py::module& m) {
     return true;
   });
   m.def("nets", []() { return gWorkspace->Nets(); });
-  m.def("run_operator_once", [](const py::bytes& op_def) {
+  m.def("run_operator_once", [](const py::bytes& op_def, bool legacy_proto=true) {
```

Comment:
hm, adding it to this API feels a bit out of place now. Is it just for test?

Reply:
Right now, it's just for test purpose.